libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/reader/AbstractInventoryReader.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.poi.ss.usermodel.DataFormatter;
 import org.apache.poi.ss.usermodel.Row;
 import org.metaeffekt.core.inventory.processor.model.*;
+import org.metaeffekt.core.inventory.processor.report.model.aeaa.AeaaInventoryAttribute;
```

Comment:
Is this import needed. Inventory should not require report.

Reply:
This import is used for the `AeaaInventoryAttribute.VULNERABILITY_REFERENCED_CONTENT_IDS` enum value for mapping the old way of writing `Referenced Content IDs` to the new one, which is `Referenced Ids`.

The issue with this is that whenever we add an attribute to the corresponding ´AbstractModelBase.Attribute´ enums, they are automatically added to the inventory when writing them out. If you check ´AeaaInventoryAttribute´ or it's Artifact-Analysis counterpart `InventoryAttribute` out, you will see a very large amount of attributes that are used in certain cases, but not always, so adding them to every inventory that is written would lead to much dead space in there.

I would also prefer moving them into the proper abstract model base implementations.